### ISSUE TITLE
Offer: Software Bill of Materials (SBOM)

### ISSUE THREAD

**Issue by BlueM** [phpughh/meeting-topics#2]
In my talk on software development and devops at the Elbphilharmonie in October 2023, I mentioned our use of SBOMs. At that time, no one in the audience was using SBOMs, although the topic has gained much attention in the last few years, especially since log4j.

While I’m not an expert on SBOMs, I could give a short talk (maybe 20 minutes) on the motivation of using SBOMs and our experiences and learnings.

Talk language: German or English

**Comment by MartinStoll** [comment]
Hi @BlueM, your offer is a bit old now... Is this topic still relevant or interesting to you, and would you be willing to give the presentation at one of the upcoming PHUGHH/SFUGHH meetups? If so, which month would work best for you? https://www.meetup.com/de-DE/phpughh/events/

**Comment by BlueM** [comment]
@MartinStoll: yes – relevant, interested and willing. November should be possible.

**Comment by MartinStoll** [comment]
Great, @BlueM , could you send me an abstract for the Meetup announcement? Then I can start getting it ready...
